paper-ieee-trans.tex

```diff
@@ -412,8 +413,7 @@ \subsubsection{SQA criteria}
 the adequate development, timely deliver and reliable operation of the produced software components.
 It emphasizes the quality requirements and best practices to be applied at the
 development phase, such as style and testing compliance or human code reviews, to protect the
-production source code versions. Furthermore, the criteria promotes the adoption of the software
-by providing minimum requirements of documentation content and stressing the usage of automated
+production source code versions. Furthermore, there are provided the minimum requirements on documentation content and it is stresed the importance of the usage of automated
```

Comment:
prefer master version

Reply:
also prefer master

Reply:
then we should change the "promotes". I don't know how provideing the minimum requirements on documentation can .. promote the adoption of software. Maybe "ensuring the compliance with the criteria" can "promote the adoption".

Reply:
master version

Reply:
the "criteria" cannot promote
maybe " compliance with the criteria"... needs reformulation.

Reply:
minumum reqs on documentation has to do with document quality which benefits adoption. Since the criteria defines these min reqs so the criteria promotes/encourages the sw adoption.